resources/views/layouts/master.blade.php

```diff
@@ -6,6 +6,11 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    @if (isset($global_vars->custom_font_embed) || isset($global_vars->custom_font_name))
+      {!! $global_vars->custom_font_embed !!}
```

Comment:
ah good call!! the only thing that changes is the kit id it looks like, so they can just give us that 😄